### PR TITLE
Added navigator in brand page

### PR DESCRIPTION
**Pull request by Nikhil-Ladha**
Signed-off-by: Nikhil-Ladha <nikhilladha1999@gmail.com>

**Description**

This PR adds a navigator to the brand page.

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [X] Yes, I signed my commits.

### CODE REVIEW

**Comment by Tanuj22** [comment]
Is it possible to have a smooth scroll to a section instead of a direct jump?

**Comment by Tanuj22** [comment]
It is present for `Image-hub` though :thinking:

**Comment by Nikhil-Ladha** [comment]
> It is present for `Image-hub` though 🤔

It is set to smooth only.

**Comment by Tanuj22** [comment]
> > It is present for `Image-hub` though thinking
> 
> It is set to smooth only.

Not working for me  

https://user-images.githubusercontent.com/42486853/103461913-bcc55d80-4d47-11eb-92da-2824acc660bb.mp4

**Comment by Nikhil-Ladha** [comment]
Yeah. It was not working 😆 
It was working for image-hub because the href was incorrect.

The issue was that link was overwriting the effect of scroll, so used `p` instead of `a` for the links.

File: src/sections/Brand/Brand-components/meshery-operator.js [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+                </Row>
+                <Row>
+                    <Col>
+                        <h3 className="layerH3 in">
+                            Logos
+                        </h3>
+                    </Col>
+                    <Row className="ImgDiv">
+                        <Col xs={12} sm={4}>
+                            <img src={MesheryOperatorDark} />
```
Comment (Tanuj22) [inline src/sections/Brand/Brand-components/meshery-operator.js:54]: Can you add `alt` here and any other place missing

Comment (Nikhil-Ladha) [reply src/sections/Brand/Brand-components/meshery-operator.js:54]: Sure.

**Review by Tanuj22 — APPROVED** [review]
LGTM

**Comment by Tanuj22** [comment]
@Nikhil-Ladha there are some conflicts

**Comment by leecalcote** [comment]
A navigator is nice. 😄

**Comment by Nikhil-Ladha** [comment]
> @Nikhil-Ladha there are some conflicts

Done.

**Review by leecalcote — COMMENTED** [review]
@Nikhil-Ladha, I'll push a commit to this real quick. We haven't exposed one of the alternative versions of the Layer5 logo: our "5" icon.

**Comment by Tanuj22** [comment]
@Nikhil-Ladha the build is failing

**Comment by leecalcote** [comment]
Love this navigator, @Nikhil-Ladha. Nice styling job, too.

**Comment by Nikhil-Ladha** [comment]
> Love this navigator, @Nikhil-Ladha. Nice styling job, too.

Making the new site better in every way possible 🙂. Glad you liked it 😎

**Pull request merged**